libs/ojsonnet/Core_jsonnet_subst.ml

```diff
@@ -0,0 +1,163 @@
+(* Yoann Padioleau
```

Comment:
What is the difference with Core_jsonnet.ml? seems like exactly the same content, can you get rid of it?